homework-submission/week3/src/todo/TodoInput.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+
+class TodoInput extends React.Component {
+  state = {
+    description: "",
+    deadline: "",
+    done: false
+  };
+
+  onInputChange = e => {
+    this.setState({
+      [e.target.name]: e.target.value
+    });
+  };
+
+  onAdd = e => {
+    e.preventDefault();
+    const value = this.state;
```

Comment:
Haha are you tired from writing the `this` keyword?! Why don't you use object destructuring? Something like:
`const { description, deadline } = this.state`